_posts/2020-03-28-learning-common-keyboard-shortuts-for-screenreaders.md

```diff
@@ -0,0 +1,37 @@
+---
+layout: post
+title: "Learning common keyboard shortcuts for screen readers"
+description: "Keyboard shortcuts are hard to learn. Here are some other ways that can help that learning."
+author: anevins12
+published: true
+categories:
+  - Assistive Technology
+further_reading:
+  - url: https://dequeuniversity.com/screenreaders/
+    title: "Screen Reader Keyboard Shortcuts and Gestures"
+    source: Deque University
+  - url: https://gist.github.com/anevins12/a9cc6548fcbab5f55c1dc3c7bbb0f4bd
+    title: "Source JSON file of the mindmap that can be imported into http://app.mindmapmaker.org/"
+    source: Github
+---
+
+Keyboard shortcuts are often presented to us in a list or a table. Lists and tables are great for some people, while others may be overwhelmed from the sheer number of commands to learn. Sometimes a more visual approach could help us learn.
+
+## Learning visually helps
+Creating a mind map of keyboard shortcuts can help learn those shortcuts easier than lists and tables. 
```

Comment:
change:
"...can help learn those..."

to
"...can help people to learn those..."